data/elan-2514-alt2.tablet

```diff
@@ -0,0 +1,22 @@
+# ELAN touchscreen/pen sensor present in the HP Envy x360 Converti0ble 15-dr1xxx
+# Width and Height are rounded up to the nearest inch.  Real measurements are:
+# Width: 13 5/8"
+# Height: 7 5/8"
```

Comment:
those values aren't really used anywhere anyway and should just reflect the marketing values. I guess this device would be marketed as 14x8 so rounding up is fine - you can drop the comment though.

Reply:
Thanks for the feedback - comment is removed in latest commit.